.github/workflows/spotless.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Check code style
```

Comment:
I am trying to standardize these names to be nouns. so I suggest "Code style check"